### ISSUE TITLE
Workbench tabs

### ISSUE THREAD

**Issue by EvgeniVasilev** [Microsoft/vscode#8944]
- VSCode Version: Code 1.3.0 (e724f269ded347b49fcf1657fc576399354e6703, 2016-07-07T16:56:12.476Z)
- OS Version: Windows_NT ia32 6.1.7601

Steps to Reproduce:

1.Open editor with opened project
2.When click  file  to open new editor only one editor (tab is opened)

**Comment by bpasero** [comment]
@EvgeniVasilev can you share more details? maybe screenshot?

**Comment by sparetire** [comment]
Same issue.
But when I drag a file to tab bar, the next file works correctly.
![bug](https://cloud.githubusercontent.com/assets/3754438/16698453/e2944a90-4580-11e6-8fd2-10f8d74d3c6a.gif)

**Comment by bpasero** [comment]
Read and understand https://code.visualstudio.com/docs/editor/tabs#_preview-mode

Closing as designed.

**Comment by misterdjules** [comment]
FWIW, I came to this issue because I found that not creating a new tab when opening a file from "quick open" was not what I wanted. I had read the latest release notes but missed any documentation on the settings mentioned in https://code.visualstudio.com/docs/editor/tabs#_preview-mode. After setting:

```
"workbench.editor.enablePreviewFromQuickOpen": false
```

in my user's settings, the behavior I get is what I expected.

I find the default behavior to be unintuitive, and the setting that allows to change that behavior to be difficult to find. I understand this is subjective, but I thought It might be useful to give feedback anyway.

In any case, thank you for the great work on Visual Studio Code, I'm a happy user!
